build.sbt

```diff
@@ -203,7 +213,8 @@ lazy val codegenCore =
     .settings(
       name := "akkaserverless-codegen-core",
       testFrameworks += new TestFramework("munit.Framework"),
-      Test / fork := false)
+      Test / fork := false,
+      Compile / PB.protoSources ++= (codegenProtoAnnotations / Compile / PB.protoSources).value)
```

Comment:
here we only need the proto annotations when compiling. We don't need to add it as a transitive dependency. Users will get it from the SDK already.